clients/desktop/src/navigation/router.tsx

```diff
@@ -83,13 +83,34 @@ const coreRoutes: Record<CorePath, ReactNode> = {
       <JoinKeysignPage />
     </ActiveVaultGuard>
   ),
+  createVaultFolder: (
+    <ActiveVaultGuard>
+      <CreateVaultFolderPage />
+    </ActiveVaultGuard>
+  ),
+  vaultFolder: (
+    <ActiveVaultGuard>
+      <CurrentVaultFolderPageProvider>
+        <VaultFolderPage />
+      </CurrentVaultFolderPageProvider>
+    </ActiveVaultGuard>
+  ),
+  manageVaultFolder: (
```

Comment:
Since these paths are identical in both clients, you can extract and put them all in sharedRoutes.